staking/programs/staking/src/lib.rs

```diff
@@ -557,15 +558,84 @@ pub mod staking {
      * The recipient of a transfer can't vote during the epoch of the transfer.
      */
     pub fn accept_split(ctx: Context<AcceptSplit>) -> Result<()> {
-        // TODO : Split vesting schedule between both accounts
+        let config = &ctx.accounts.config;
```

Comment:
don't we need to set a flag somewhere so the recipient can't vote? I don't see that in the method implementation.

Reply:
In the new_stake_account_metadata initialization